lib/school.rb

```diff
@@ -0,0 +1,27 @@
+class School
+  attr_reader :start_time,
+              :hours_in_school_day,
+              :student_names
+
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(name)
+    student_names << name
+  end
+
+  def end_time
+    @start_time += @hours_in_school_day
```

Comment:
In order to use math, you will first need to convert the this string to an integer; something like `start_time.to_i`